### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by otavio**
The update was simple and just had to fix few warnings.

 - lazy_static
 - rand
 - regex

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

### CODE REVIEW

**Comment by otavio** [comment]
Any idea why it failed on AppVeyor?

File: Cargo.toml [hunk 1]
```diff
@@ -20,10 +20,10 @@ path = "src/main.rs"
 doc = false
 
 [dependencies]
-rand = "^0.3.14"
+rand = "0.5.5"
 http-muncher = "^0.3"
-regex = "^0.2.2"
-lazy_static = "^0.2"
+regex = "1.0.5"
```
Comment (lipanski) [inline Cargo.toml]: please preserve the `^` (here and everywhere else), so we can still be flexible about the allowed version. see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html for more details.

**Comment by otavio** [comment]
I made the changes you requested. Is there anything else pending?

**Comment by lucab** [comment]
@lipanski ping, mind having a look and hopefully merge this?

**Comment by lipanski** [comment]
merged!

@otavio thanks for your contribution & sorry for the delay.

**Comment by lucab** [comment]
@lipanski thanks a lot! Do you have any idea what's going on with AppVeyor?

**Comment by lipanski** [comment]
@lucab https://github.com/lipanski/mockito/issues/41 there's this ticket to track the issue but I didn't have time to investigate. I assume it's the http parser behaving in a different way on Windows machines.

**Pull request merged**